images/router/haproxy/conf/haproxy-config.template

```diff
@@ -39,17 +39,34 @@ defaults
   option httplog
   log global
 {{ end }}
+
+{{ if (matchPattern "[1-9][0-9]*(us|ms|s|m|h|d)?" (env "ROUTER_DEFAULT_CONNECT_TIMEOUT" "")) }}
```

Comment:
I agree that they should all have the validation, but you also need to put the default on the env call here. Otherwise this will always evaluate as false unless there is a set value. This is true for all the timeouts